src/Service/Mapping.php

```diff
@@ -266,4 +262,18 @@ public function updateMapping(string $name, array $mappings, string $type): void
         $endpoint->setType($this->getTypePath($type));
         $this->elasticaClient->requestEndpoint($endpoint);
     }
+
+    /** @param array<string, array<string, mixed>> $properties */
+    private function addCopyToAllField(array &$properties): void
+    {
+        foreach ($properties as $name => &$options) {
+            if (\in_array($options['type'] ?? null, ['text', 'keyword'], true)) {
+                $options['copy_to'] = \array_unique(\array_merge(['_all'], $options['copy_to'] ?? []));
+                continue;
+            }
+            if (\in_array($options['type'] ?? null, ['nested'], true)) {
```

Comment:
```suggestion
            if (\in_array(($options['type'] ?? null), ['nested'], true)) {
```

Reply:
Same structure as the if test above, easier to extend. Discussed this also with Mathieu